components/kyma-environment-broker/internal/auditlog/config.go

```diff
@@ -1,9 +1,68 @@
 package auditlog
 
+import (
+	"errors"
+	"text/template"
+
+	"github.com/gobuffalo/packr"
+
+	"github.com/Masterminds/semver"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/cls"
+	"github.com/sirupsen/logrus"
+)
+
+//go:generate packr -v
+
 type Config struct {
 	URL           string `envconfig:"APP_AUDITLOG_URL"`
 	User          string `envconfig:"APP_AUDITLOG_USER"`
 	Password      string `envconfig:"APP_AUDITLOG_PASSWORD"`
 	Tenant        string `envconfig:"APP_AUDITLOG_TENANT"`
 	EnableSeqHttp bool   `envconfig:"APP_AUDITLOG_ENABLE_SEQ_HTTP"`
 }
+
+type Overrides struct {
+	Host         string
+	Port         string
+	Path         string
+	HttpPlugin   string
+	ClsOverrides *cls.ClsOverrides
+	Config       Config
+}
+
+func getExtraConf119(log logrus.FieldLogger) (*template.Template, error) {
```

Comment:
Function name can be more descriptive. Something like getExtraConfForKyma1_19

Reply:
done.